modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes.jsp

```diff
@@ -87,6 +87,7 @@ AssignScopesDisplayContext assignScopesDisplayContext = (AssignScopesDisplayCont
 				</div>
 
 				<aui:input id="impliedScopeAliases" name="scopeAliases" type="hidden" />
+				<aui:input id="newInputField" name="newInputField" type="hidden" />
```

Comment:
This line adds a form input. Please check:

    * The input is validated on the processing action

    * The backend is protected agains value forging

    

    Refer to documentation: